tests/test_local_setup.py

```
@@ -0,0 +1,25 @@
+    environment.update(dotenv_values(root / ".env.example"))
+    result = subprocess.run(
+        [sys.executable, "main.py", "--help"],
+        cwd=root,
+        env=environment,
+        capture_output=True,
+        text=True,
+        timeout=30,
+        check=False,
+    )
+    assert result.returncode == 0, result.stderr
+    assert "server" in result.stdout and "worker" in result.stdout
```

Ruff (0.16.3)

[warning] Assertion should be broken down into multiple parts

Break down assertion into multiple parts

(PT018)